### DISCUSSION TITLE
Autoraise or Autofocus

### DISCUSSION THREAD

**Discussion by Mirage-Panda** [asmvik/yabai#2505] [Q&A] [answered]
In the wiki, I noticed that focus_follows_mouse had 2 options: autofocus and autoraise. I don't get the difference between raising and focusing—how are they different?

**Comment by dominiklohmann** [comment, marked as answer]
`autofocus` focuses, `autoraise` focuses and raises the window. That is, if you have overlapping windows and you use `autofocus`, you may be able to focus a window that's not entirely visible because it stays obscured by another window that is not focused.

**Reply by Mirage-Panda** [reply]
So basically if I have 2 overlapping windows and I use autofocus, the second window doesn't get put on top. Is this correct?
